ovc5/firmware/petalinux/.gitignore

```diff
@@ -11,3 +11,5 @@ pre-built/linux/
 *.xsa
 /components/plnx_workspace
 /components/yocto
+project-spec/
```

Comment:
We probably don't want to ignore everything that is under here since those are the bitbake recipes we need to customized fsbl / u-boot / kernel / device tree

Reply:
Fixed this! Now it just directly ignores the `.bit` files that gets created when importing the vivado export.